### PR TITLE
* [ADD] Adding new module : project_issue_code

### PR DESCRIPTION
**Pull request by mikevhe18**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please use latest README template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst and check Travis errors

File: project_issue_code/__init__.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from . import models
+from openerp import SUPERUSER_ID
+
+
+def create_code_equal_to_id(cr):
+    """
```
Comment (pedrobaeza) [inline project_issue_code/__init__.py]: Please put these hooks into a file like the conventions: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#installation-hooks

**Comment by mikevhe18** [comment]
@pedrobaeza thanks for the review, I'll update it soon

File: project_issue_code/README.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    * Project issue sequence
+    
+Field "issue_code" is readonly and its value will be filled in accordance with the sequence.
+For the first time installation, the sequence is automatically configured.
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
```
Comment (tafaRU) [inline project_issue_code/README.rst:26]: Please replace below the placeholders by their corresponding values.

File: project_issue_code/__init__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Michael Viriyananda
+#    Copyright 2016 OpenSynergy Indonesia
+#
+#    This program is free software: you can redistribute it and/or modify
```
Comment (tafaRU) [inline project_issue_code/__init__.py]: Please use short header like the conventions for all py files: https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1-L3

Thank you in advance!

File: project_issue_code/README.rst (hunk 2 above)
Comment (mikevhe18) [inline project_issue_code/README.rst:26]: Okay, i forgot to replace it. Thanks for the review.

**Comment by mikevhe18** [comment]
#150

File: project_issue_code/README.rst [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Michael Viriyananda <viriyananda.michael@gmail.com>
+
+Inspired By
+------------
```
Comment (dreispt) [inline project_issue_code/README.rst]: No need to make this a top header.

File: project_issue_code/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+{
+    "name": "Project Issue Code",
+    "version": "8.0.1.0.0",
+    'author': 'Michael Viriyananda,Odoo Community Association (OCA)',
+    "category": "Project Management",
+    'summary': 'Adding Field Code For Project Issue',
+    'website': 'http://github.com/mikevhe18',
+    "license": "AGPL-3",
+    'description': """
```
Comment (dreispt) [inline project_issue_code/__openerp__.py]: remove the description key, or it will "silence" the README

File: project_recalculate/models/project_project.py [hunk 6]
```diff
@@ -56,12 +56,12 @@ def project_recalculate(self):
...
-            if (project.calculation_type == 'date_begin'
-                    and not project.date_start):
+            if (project.calculation_type == 'date_begin' and not
+                    project.date_start):
                 raise Warning(_("Cannot recalculate project because your "
                                 "project don't have date start."))
-            if (project.calculation_type == 'date_end'
-                    and not project.date):
+            if (project.calculation_type == 'date_end' and not
+                    project.date):
```
Comment (dreispt) [inline project_recalculate/models/project_project.py:64]: Please revert these changes - they are unrelated to the PR.

**Comment by dreispt** [comment]
Review: other than some minor issues, looks good.

File: project_recalculate/models/project_project.py (hunk 6 above)
Comment (dreispt) [inline project_recalculate/models/project_project.py:64]: I guess this is related to https://github.com/OCA/project/pull/150, merged in the meanwhile.

**Comment by mikevhe18** [comment]
@tafaRU @pedrobaeza @rafaelbn @dreispt i've fixed some minor issues. Would you kindly review this PR again?

**Comment by dreispt** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Comment by dreispt** [comment]
@mikevhe18 Was this your first OCA contribution?

**Comment by pedrobaeza** [comment]
@mikevhe18 I didn't find your Twitter account for mentioning you in a tweet. I put `mikevhe18`

**Comment by mikevhe18** [comment]
@dreispt I have contributed in other projects but this is the first time my PR successfully merged
@pedrobaeza my twitter account is "@MichaelViriya"

**Comment by dreispt** [comment]
Congratulations then! :tada:
